plugins/org.yakindu.sct.model.stext/src/org/yakindu/sct/model/stext/validation/STextValidator.xtend

```diff
@@ -791,7 +791,15 @@ class STextValidator extends AbstractSTextValidator implements STextValidationMe
 			}
 		}
 	}
-	
+	@Check
+	def void checkImportNotCyclicsct(Statechart sct) {
```

Comment:
Why is there an "sct" in the name? Looks strange.

Reply:
renamed

Reply:
I meant checkImportNotCyclic**sct** => checkImportNotCyclic

Reply:
:+1: